src/main/java/com/exasol/RepositoryHandler.java

```diff
@@ -59,19 +62,24 @@ private void validatePlatforms(final String branch) {
                     platform, this.validationReport);
             platformValidator.validate();
         }
-        LOGGER.info(() -> "Validation completed.");
     }
 
     /**
      * Release the project.
+     * 
+     * @return release report
      */
-    public void release() {
+    public ReleaseReport release() {
         LOGGER.info(() -> "Release started.");
```

Comment:
Maybe also log `Release finished` at the end of this method?

Reply:
I removed it because it logs `Release finished` even if it actually failed. I thought it was misleading. The report summary contains this information anyway